src/importexport/midi/internal/midiimport/importmidi_simplify.cpp

```diff
@@ -62,7 +62,7 @@ bool areDurationsEqual(
         sum += ReducedFraction(d.second.fraction()) / d.first;
     }
 
-    return desiredLen == desiredLen;
+    return sum == desiredLen;
```

Comment:
See https://github.com/musescore/MuseScore/pull/7627#pullrequestreview-604922180 where this had come up before, and also #7179 (for 3.x), this change results in failing mtests (on 3.x at least).
Not sure why it doesn't on master, maybe the corresponding tests are disabled?

Reply:
#7179 has just been merged, if you like integrate that one into your PR, else I'd port it over (it basically removes that stuff)

Reply:
See #8849